### PR TITLE
Move release notes to changelog

### LINKED ISSUE #837: Update Release notes for the v0.6.0a10 release

### PR DESCRIPTION
**Pull request by kozbo**
closes #837 
and replaces PR #838

### CODE REVIEW

File: doc/source/changelog.rst [hunk 1]
```diff
@@ -2,6 +2,62 @@
 Changelog
 #########
 
+Schema Release `v0.6.0a10`
```
Comment (david4096) [inline doc/source/changelog.rst]: Double backticks here to match the style elsewhere.

**Comment by david4096** [comment]
Thanks @kozbo a ton of stuff in this release!

File: doc/source/changelog.rst [hunk 2]
```diff
@@ -2,6 +2,62 @@
...
+  * `GET /expressionlevels/{id}`
+ - Added support for BigWig files in a new Continuous Data object #802 Adds the following new endpoints:
+  - Continuous Data: `POST /continuoussets/search`
+  - Continuous Data: `GET /continuoussets/{id}`
+  - Continuous Data: `POST /continuous/search`
+ * Add deep set/get attr to protocol module #816
+ * Changed ontology term “id” to “term_id” #805 Impacts the following message types:
+  * OntologyTerm
+ * Replaced info fields with rich type Attributes fields #700 Impacts the following message types:
+  * TranscriptEffect
```
Comment (david4096) [inline doc/source/changelog.rst]: These can maybe be collapsed to a single wrapped line?

File: doc/source/changelog.rst [hunk 3]
```diff
@@ -2,6 +2,62 @@
...
+ * Add ability to list and join peer server networks #760 Adds the following new endpoints:
+  * Peer: `POST /peers/list`
+  * Peer: `POST /peers/announce`
+  * Peer: `GET /info`
+ * Replace NCBI taxon ID integer with ontology term (#699) Impacts the following message types:
+  * Reference
+  * ReferenceSet
+  
+Documentation:
+ * Add instructions for viewing the UML diagram #835
```
Comment (david4096) [inline doc/source/changelog.rst]: Thanks for putting the issue numbers, it is nice to give the hyperlinks with them if you can.

Comment (kozbo) [inline doc/source/changelog.rst]: I  think that they are easier to read this way.

File: doc/source/changelog.rst (hunk 2 above)
Comment (kozbo) [inline doc/source/changelog.rst]: I think that they are easier to read in list form, so I would like to leave them this way.

**Comment by kozbo** [comment]
closing in favor of #860

**Pull request closed without merging**